package.json

```diff
@@ -3,6 +3,14 @@
   "version": "1.1.13",
   "description": "Streams3, a user-land copy of the stream library from Node.js v0.11.x",
   "main": "readable.js",
+  "files": [
```

Comment:
no. you should include the readme tests and examples.

Being able to debug and understand node_modules locally is more important then saving bytes.